stream-chat-android-client/src/main/java/io/getstream/chat/android/client/notifications/LoadNotificationDataWorker.kt

```diff
@@ -61,13 +61,16 @@ internal class LoadNotificationDataWorker(
             )
 
             val result = getChannel.zipWith(getMessage).await()
-            if (result.isSuccess) {
-                val (channel, message) = result.data()
-                ChatClient.displayNotification(channel = channel, message = message)
-                Result.success()
-            } else {
-                logger.e { "Error while loading notification data: ${result.error().stringify()}" }
-                Result.failure()
+            when (result) {
+                is io.getstream.chat.android.client.utils.Result.Success -> {
```

Comment:
I think we don't need the whole import path here.

Reply:
Unfortunately we need it as below we are returning Result from `androidx.work` package